completion/available/gradle.completion.bash

```diff
@@ -66,10 +68,11 @@ __gradle-generate-script-cache() {
 	local cache_ttl_mins=${GRADLE_CACHE_TTL_MINUTES:-30240}
 	local script_exclude_pattern=${GRADLE_COMPLETION_EXCLUDE_PATTERN:-"/(build|integTest|out)/"}
 
-	if [[ ! $(find $cache_dir/$cache_name -mmin -$cache_ttl_mins 2> /dev/null) ]]; then
+	if [[ ! $(find "$cache_dir/$cache_name" -mmin "-$cache_ttl_mins" 2> /dev/null) ]]; then
 		# Cache all Gradle scripts
-		local gradle_build_scripts=$(find $project_root_dir -type f -name "*.gradle" -o -name "*.gradle.kts" 2> /dev/null | grep -E -v "$script_exclude_pattern")
-		printf "%s\n" "${gradle_build_scripts[@]}" > $cache_dir/$cache_name
+		local gradle_build_scripts
+		gradle_build_scripts=$(find "$project_root_dir" -type f -name "*.gradle" -o -name "*.gradle.kts" 2> /dev/null | grep -E -v "$script_exclude_pattern")
```

Comment:
This is not an issue introduced in this PR, but I suspect the intent would be

```suggestion
		gradle_build_scripts=$(find "$project_root_dir" -type f \( -name "*.gradle" -o -name "*.gradle.kts" \) 2> /dev/null | grep -E -v "$script_exclude_pattern")
```

`.kts` seems to be the filename extension of a Kotlin file, so it must be a regular file.

Reply:
could be, but that's for the original contributor to fix. under a separate PR.

Reply:
Went over the manpage, there's an `-or` logical operator, and there's a `-o` in the examples (that doesn't suggest a logical or), but no `-o` in the list of parameters documented. I'm sticking with not changing that line.

Reply:
man page from macOS 15.3

Reply:
Thanks. I'm not sure why macOS man page doesn't clearly mention the `-o` operator, but the `find` utility is anyway supposed to follow POSIX, and POSIX defines the `-o` operator as logical OR.